src/net_processing.cpp

```diff
@@ -334,6 +339,9 @@ struct CNodeState {
       *     and set a shorter timeout, HEADERS_RESPONSE_TIME seconds in future.
       *     If their best known block is still behind when that new timeout is
       *     reached, disconnect.
+      *
+      * EXTRA_PEER_CHECK_INTERVAL: after each interval, if we have too many outbound peers,
+      * drop to the outbound one that least recently announced us a new block.
```

Comment:
`* drop to the outbound one that least recently announced us a new block.` -> 
`* drop the outbound one that least recently announced us a new block.` ("to" does not belong here)